src/js/Maze.js

```diff
@@ -51,6 +51,42 @@ class Maze {
             return index ===cells.length ? false : true;
         }
     }
+
+    /**
+     * Calculate distance of all maze cells from cell at given index,
+     * then use color to visualize distance.
+     *
+     * @param {number} cellIdx - Index of selected start cell.
+     */
+    colorByDistanceFromCell(cellIdx) {
```

Comment:
I know another purpose of this function was to help us find the shortest path. I also think functions on the maze should be responsible for processing/reporting INFORMATION about the maze, not for rendering it. Therefore I wonder if it would make sense (not now necessarily, but soon) to
(1) Change its name to something like "getDistancesFromCellDictionary"
(2) Have it return a dictionary mapping cells (or cell coordinates) to the distance from the cell
(3) Then, call that function and do the coloring logic in `index.js`, since I don't think this kind of styling action really belongs on the maze object anyway